rust/agama-locale-data/src/timezone_part.rs

```diff
@@ -42,22 +42,22 @@ impl TimezoneIdParts {
         self.timezone_part
             .iter()
             .map(|part| (part.id.clone(), part.names.name_for(language)))
-            .for_each(|(time_id, names)| -> () {
+            .for_each(|(time_id, names)| {
                 // skip missing translations
                 if let Some(trans) = names {
                     res.insert(time_id, trans);
                 }
             });
-        return res;
+        res
     }
 
     fn translate_timezone(&self, mapping: &HashMap<String, String>, timezone: &str) -> String {
         timezone
-            .split("/")
+            .split('/')
             .map(|tzp| {
                 mapping
                     .get(&tzp.to_string())
-                    .expect(format!("Unknown timezone part {tzp}").as_str())
+                    .unwrap_or_else(|| panic!("Unknown timezone part {tzp}"))
```

Comment:
JFYI: I liked the original more so I ran clippy and found the reason for the change: the `format!` would always be called.

Reply:
Yes, same with me. :)